ui/app/AppLayouts/CommunitiesPortal/views/CommunitiesGridView.qml

```diff
@@ -111,7 +111,7 @@ StatusScrollView {
         StatusBaseText {
             visible: !root.searchLayout && popularRepeater.count
             Layout.topMargin: 20
-            text: qsTr("Popular")
+            text: qsTr("All")
```

Comment:
needs a little context, sth like "all communities"

Reply:
Translators will see the context of the file "CommunitiesGridView", I think it will be enough :thinking:

Reply:
Right, added context for translators